github.go

```diff
@@ -157,3 +201,57 @@ func (g *GitHubPullRequest) comment() ([]*github.PullRequestComment, error) {
 	}
 	return comments, nil
 }
+
+// ---
+// GitHub PullRequest Review API Implementation
+// ref: https://github.com/google/go-github/issues/495
+
+const (
+	mediaTypePullRequestReview = "application/vnd.github.black-cat-preview+json"
+)
+
+type Review struct {
+	Body     *string          `json:"body,omitempty"`
+	Event    *string          `json:"event,omitempty"`
+	Comments []*ReviewComment `json:"comments,omitempty"`
+}
+
+type ReviewComment struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type ReviewComment should have comment or be unexported